src/backend/aspen/database/models/accessions.py

```diff
@@ -5,13 +5,61 @@
 from dataclasses import dataclass
 from typing import Optional, Type
 
-from sqlalchemy import Column, ForeignKey, Integer, String, UniqueConstraint
-
+import enumtables
+from sqlalchemy import (
+    Column,
+    ForeignKey,
+    Integer,
+    PrimaryKeyConstraint,
+    String,
+    UniqueConstraint,
+)
+
+from aspen.database.models.base import base
 from aspen.database.models.entity import Entity, EntityType
+from aspen.database.models.enum import Enum
 from aspen.database.models.workflow import Workflow, WorkflowStatusType, WorkflowType
 
 
-class Accession(Entity):
+class AccessionType(enum.Enum):
+    GISAID_ISL = "GISAID_ISL"
+    GENBANK = "GENBANK"
+
+
+_AccessionTypeTable = enumtables.EnumTable(
+    AccessionType,
+    base,
+    tablename="accession_types",
+)
+
+
+class Accession(base):  # type: ignore
+    """An accession for a single sample."""
+
+    __tablename__ = "accessions"
+    __table_args__ = (
+        PrimaryKeyConstraint("sample_id", name="pk_accessions_sample_id"),
```

Comment:
Sample ID isn't a primary key! We do need the unique key below but don't need this pk constraint.

Reply:
yes, this was leftover from the previous way I did the table, thanks for catching this!